db/migrate/20160927230136_enable_fk_message_digests_nodes.rb

```diff
@@ -0,0 +1,21 @@
+class EnableFkMessageDigestsNodes < ActiveRecord::Migration
+
+  class Bag < ActiveRecord::Base
+  end
+
+  class MessageDigest < ActiveRecord::Base
+    belongs_to :bag
+  end
+
+  def up
```

Comment:
Given a starting state of a mysql database that has successfully run the migrations currently in develop, this migration produces an incorrect state (duplicate foreign keys):

```
  add_foreign_key "message_digests", "nodes", name: "message_digests_nodes_id_fk", on_update: :cascade
  add_foreign_key "message_digests", "nodes", on_update: :cascade
```

I imagine removing the specific name for this FK will then lead to an error about duplicate constraint names.